docs/user/geoserver.rst

```diff
@@ -0,0 +1,92 @@
+GeoMesa GeoServer Plugins
+=========================
+
+Background
+----------
+
+When you install the GeoMesa plugins for GeoServer, you get access to the
```

Comment:
OK, I've rewritten the chapter to have general instructions on setting up a GeoMesa data source and publishing a layer.